scripts/cv_api_master.py

```diff
@@ -1,127 +1,477 @@
+                    "generate": ["keras_cv.layers.VectorizedBaseImageAugmentationLayer"],
+                },
+            ],
         },
         {
-            "path": "equalization",
-            "title": "Equalization layer",
-            "generate": ["keras_cv.layers.Equalization"],
+            "path": "decoder/",
+            "title": "Decoder Layers",
+            "toc": True,
+            "children": [
+                {
+                    "path": "feature_pyramid",
+                    "title": "Feature Pyramid layer",
+                    "generate": ["keras_cv.layers.FeaturePyramid"]
+                },
+                {
+                    "path": "spatial_pyramid_pooling",
+                    "title": "Spatial Pyramid Pooling layer",
+                    "generate": ["keras_cv.layers.SpatialPyramidPooling"]
+                },
+            ]
+        },
+        {
+            "path": "embedding/",
+            "title": "Embedding layers",
+            "toc": True,
+            "children": [
+                {
+                    "path": "add_relative_positional_embedding",
+                    "title": "Add Relative Positional Embedding layer",
+                    "generate": [
+                        "keras_cv.layers.AddRelativePositionalEmbedding"
+                    ]
+                },
+                {
+                    "path": "multi_head_attention_with_relative_pe",
+                    "title": "Mlti-Head Relative Positional Embedding layer",
+                    "generate": [
+                        "keras_cv.layers.MultiHeadAttentionWithRelativePE"
+                    ]
+                },
+                {
+                    "path": "overlapping_patching_and_embedding",
+                    "title": "Overlapping Patching and Embedding layer",
+                    "generate": [
+                        "keras_cv.layers.OverlappingPatchingAndEmbedding"
+                    ]
+                },
+                {
+                    "path": "patching_and_embedding",
+                    "title": "Patching and Embedding layer",
+                    "generate": [
+                        "keras_cv.layers.PatchingAndEmbedding"
+                    ]
+                },
+                {
+                    "path": "vitdet_patching",
+                    "title": "ViT Det Patching and Embedding layer",
+                    "generate": [
+                        "keras_cv.layers.ViTDetPatchingAndEmbedding"
+                    ]
+                },
+                {
+                    "path": "window_partitioning",
+                    "title": "Window Partitioning layer",
+                    "generate": [
+                        "keras_cv.layers.WindowPartitioning"
+                    ]
+                },
+            ]
+        },
+        {
+            "path": "modelling/",
+            "title": "Modelling layers",
+            "toc": True,
+            "children": [
```

Comment:
You can remove the sub-section and just put all the layers directly under `modeling`.

Reply:
So will it be something like `MODELING_MASTER`?